docs/source/deployment.rst

```diff
@@ -54,6 +60,14 @@ To speed up the Terraform run, the module name may be specified with the ``targe
   $ python manage.py build --target kinesis       # tf_stream_alert_kinesis module
   $ python manage.py build --target stream_alert  # tf_stream_alert module
 
+To apply specific terraform changes, go to ``terraform`` folder and run ``terraform apply`` with ``-target`` option manually:
+
+.. code-block:: bash
+
+  $ cd terraform
```

Comment:
I'm not sure I'd like to highlight this unless we absolutely have to.

Is there something that `python manage.py list-targets` wouldn't allow you to target with `python manage.py build --target ...`?